docs/usage/create-delta-lake-table.md

```diff
@@ -0,0 +1,26 @@
+# Creating a Delta Lake Table
+
+This section explains how to create a Delta Lake table.
+
+You can easily write a DataFrame to a Delta table.
+
+```python
+import deltalake as dl
+from deltalake.writer import write_deltalake
```

Comment:
We expose write_deltalake also on the deltalake module directly. Also in other docs we import directly from there, maybe do it here as well for consistency

Reply:
I get this error: `cannot import name 'DeltaLake' from 'deltalake'`

You sure `write_deltalake` is exposed on `deltalake`?

Reply:
I had a typo. You're right.  Updated.